src/magichome-interface/Transport.ts

```diff
@@ -96,9 +101,12 @@ export class Transport {
   }
 
   async send(buffer: any, useChecksum = true, _timeout = 2000) {
+    this.log.warn(`[IGOR] send (adding to queue) data to ${this.host}`);
```

Comment:
Why is your name in the logs? If you'd like to comment your name next to the code block that's fine. Please remove any personally identifiable information from logs.

Reply:
I added my name, so it would be easy to spot for removal. It's been removed :)